clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -75,6 +75,11 @@ using namespace clang::driver::tools;
 using namespace clang;
 using namespace llvm::opt;
 
+static bool hasSYCLCudaCompatFlag(const ArgList &Args) {
```

Comment:
Pedantic naming suggestion comment that you should feel free to ignore.
```suggestion
static bool isSYCLCudaCompatEnabled(const ArgList &Args) {
```